docs/user/sending-notifications.rst

```diff
@@ -0,0 +1,115 @@
+Sending Notifications
+=====================
+
+Notifications can be created using the ``notify`` signal. Eg:
+
+.. code-block:: python
+
+    from django.contrib.auth import get_user_model
+    from swapper import load_model
+
+    from openwisp_notifications.signals import notify
+
+    User = get_user_model()
+    Group = load_model("openwisp_users", "Group")
+    admin = User.objects.get(email="admin@admin.com")
+    operators = Group.objects.get(name="Operator")
+
+    notify.send(
+        sender=admin,
```

Comment:
Why is this example not mentioning `type`, ``url`` and ``email_subject``?
Those arguments seem the arguments we use most in OpenWISP, so not having them in this main example is puzzling to me.